app/pages/BeelieversAuction/MintInfo.tsx

```diff
@@ -198,9 +198,19 @@ function MintAction({ isWinner, doRefund, hasMinted, setNftId }: MintActionProps
 				kioskCapId = kioskInfo.kioskCapId;
 			}
 
+			const tx = createMintTx(kioskId, kioskCapId, beelieversMint, beelieversAuction.auctionId);
+
+			// Set sender before dry run check
+			tx.setSender(account.address);
+
+			// Do dry run check BEFORE sending to wallet
+			console.log("Running pre-transaction validation...");
+			await client.dryRunTransactionBlock({
+				transactionBlock: await tx.build({ client }),
+			});
```

Comment:
is there a param / option in `executeTransaction` to return dry run effects? Because, today, the dry run is already happening - so with this code we will be doing it 2 times, right?